src-js/experiments/DevHubDocsNav.js

```diff
@@ -0,0 +1,17 @@
+
+$(() => {
+  // https://app.optimizely.com/v2/projects/16812830475/experiments/20616121296/variations
+  window.OptimizelyClient.getVariationName({
+    experimentKey: 'dd_merge_devhub_docs_experiment_test',
+    groupExperimentName: 'q3_fy22_docs_disco_experiment_group_test'
+  }).then(variation => {
+    if (variation === "treatment") {
+      $(".dd-global-nav--links").show();
+      $(".dd-devhub-docs-nav-link").click(function(event) {
+        event.preventDefault();
+        window.AnalyticsClient.trackAction('docs-devhub-nav-merge-link-clicked', {link: this.href})
+        window.location = this.href
```

Comment:
curious why do we need `event.preventDefault();` and `window.location = this.href` ?

Reply:
I made the assumption that the <a> was navigating away from the page before the tracking invocation could execute. I will double check this is the case, though.

Reply:
Update - this was a bad assumption - it was unnecessary to preventDefault. I have updated this.